### PR TITLE
Add Vercel integration docs

### PR DESCRIPTION
**Pull request by pinkerton**
### What does this PR do?

Adds docs for our upcoming Vercel integration

Screen Shot 2021-07-01 at 10 49 32 AM

### CODE REVIEW

File: vercel/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+To set up the Vercel-Datadog integration:
+
+* Generate a [Datadog API key][6]
+* Configure the logs integration through the [Vercel Marketplace][7]
+
+
+### Installation
+
+To install the Vercel check on your host:
```
Comment (cswatt) [inline vercel/README.md:23]: are there supposed to be instructions following?

Comment (pinkerton) [reply vercel/README.md]: The instructions are above - this was a copy paste error

**Review by cswatt — APPROVED** [review]

File: vercel/manifest.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Vercel",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
```
Comment (ChristineTChen) [inline vercel/manifest.json:22]:
```suggestion
  "is_public": true,
```

enable docs

**Review by ChristineTChen — CHANGES_REQUESTED** [review]

File: flume/datadog_checks/flume/data/conf.yaml.example [hunk 3]
```diff
@@ -20,7 +20,7 @@ init_config:
     ##                       jvm.gc.major_collection_time
     ## The default value is false to ensure backward compatibility.
     #
-    # new_gc_metrics: false
```
Comment (coignetp) [inline flume/datadog_checks/flume/data/conf.yaml.example:23]: Can you revert this change?

File: hbase_master/datadog_checks/hbase_master/data/conf.yaml.example [hunk 4]
```diff
@@ -148,7 +148,7 @@ init_config:
     ##                       jvm.gc.major_collection_time
     ## The default value is false to ensure backward compatibility.
     #
-    # new_gc_metrics: false
```
Comment (coignetp) [inline hbase_master/datadog_checks/hbase_master/data/conf.yaml.example:151]: Can you revert this change?

File: hbase_regionserver/datadog_checks/hbase_regionserver/data/conf.yaml.example [hunk 5]
```diff
@@ -148,7 +148,7 @@ init_config:
     ##                       jvm.gc.major_collection_time
     ## The default value is false to ensure backward compatibility.
     #
-    # new_gc_metrics: false
```
Comment (coignetp) [inline hbase_regionserver/datadog_checks/hbase_regionserver/data/conf.yaml.example:151]: Can you revert this change?

File: open_policy_agent/manifest.json [hunk 6]
```diff
@@ -16,7 +16,8 @@
   "categories": [
     "security",
     "containers",
-    "configuration & deployment"
```
Comment (coignetp) [inline open_policy_agent/manifest.json:19]: Can you revert this change?

File: resin/datadog_checks/resin/data/conf.yaml.example [hunk 7]
```diff
@@ -20,7 +20,7 @@ init_config:
     ##                       jvm.gc.major_collection_time
     ## The default value is false to ensure backward compatibility.
     #
-    # new_gc_metrics: false
```
Comment (coignetp) [inline resin/datadog_checks/resin/data/conf.yaml.example:23]: Can you revert this change?

File: .azure-pipelines/all.yml [hunk 8]
```diff
@@ -153,6 +153,9 @@ jobs:
     - checkName: upsc
       displayName: UPSC
       os: linux
+    - checkName: vercel
```
Comment (coignetp) [inline .azure-pipelines/all.yml:156]: Since vercel is a tile-only integration, you can remove it from the ci

**Review by coignetp — CHANGES_REQUESTED** [review]

**Review by coignetp — CHANGES_REQUESTED** [review]
Can you delete the following files? Since it's a tile integration you don't need those, it should fix the CI
```
vercel/MANIFEST.in
vercel/requirements-dev.txt
vercel/requirements.in
vercel/setup.py
vercel/tox.ini
```

**Review by ofek — APPROVED** [review]
